### PR TITLE
ignore cluster access with save flag

### PR DESCRIPTION
**Pull request by sheharyaar**

### CODE REVIEW

**Comment by sheharyaar** [comment]
fixes #386

File: install/install.go [hunk 1]
```diff
@@ -236,7 +241,12 @@ func K8sInstaller(c *k8s.Client, o Options) error {
 			},
 		}
 		if _, err := c.K8sClientset.CoreV1().Namespaces().Create(context.Background(), &newns, metav1.CreateOptions{}); err != nil {
-			return fmt.Errorf("failed to create namespace %s: %+v", ns, err)
+			if o.Save {
+				fmt.Printf("⚠️\tfailed to create namespace %s: %+v\n", ns, err)
```
Comment (nyrahul) [inline install/install.go:245]: I think if `save` is enabled then karmor should not be creating any k8s resources including namespace ... 

The logic should be to bypass namespace create if `save` is attempted.

@daemon1024 WDYT?

**Review by nyrahul — CHANGES_REQUESTED** [review]

File: install/install.go (hunk 1 above)
Comment (daemon1024) [reply install/install.go:245]: That's correct.

Comment (sheharyaar) [reply install/install.go:245]: Ok, will do the required changes

**Review by daemon1024 — APPROVED** [review]
LGTM, Thanks for the quick handling.

**Pull request merged**